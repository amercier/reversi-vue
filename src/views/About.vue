<template>
  <main class="about"><!-- TODO Implement about page --></main>
</template>
