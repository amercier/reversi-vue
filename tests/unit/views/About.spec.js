import { shallowMount } from '@vue/test-utils';
import About from '@/views/About.vue';

describe('About.vue', () => {
  it('renders without crashing', () => {
    shallowMount(About);
  });

  it('renders its components', () => {
    const about = shallowMount(About);
    expect(about.html()).toMatchSnapshot();
  });
});
