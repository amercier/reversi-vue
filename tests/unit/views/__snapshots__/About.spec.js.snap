// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About.vue renders its components 1`] = `
<main>
  <div
    class="about"
  >
    <h1
      class="about-title"
    >
      Reversi
    </h1>
     
    <p
      class="about-paragraph"
    >
      
      Created by
      
      <a
        class="about-link"
        href="https://github.com/amercier"
        rel="nofollow noopener noreferrer"
        target="_blank"
      >
        Alex Mercier
      </a>
      
      for
      
      <a
        class="about-link"
        href="https://www.backmarket.fr/"
        rel="nofollow noopener noreferrer"
        target="_blank"
      >
        Back Market
      </a>
      .
    
    </p>
     
    <h2
      class="about-subtitle"
    >
      Development
    </h2>
     
    <p
      class="about-paragraph"
    >
      
      This game was developed with
      
      <a
        class="about-link"
        href="https://vuejs.org/"
        rel="nofollow noopener noreferrer"
        target="_blank"
      >
        Vue.js 2
      </a>
      , using
      
      <a
        class="about-link"
        href="https://cli.vuejs.org/"
        rel="nofollow noopener noreferrer"
        target="_blank"
      >
        Vue CLI 3
      </a>
      
      and
      
      <a
        class="about-link"
        href="https://vuex.vuejs.org/"
        rel="nofollow noopener noreferrer"
        target="_blank"
      >
        Vuex
      </a>
      .
    
    </p>
     
    <p
      class="about-paragraph"
    >
      
      Source code is available on
      
      <a
        class="about-link"
        href="https://github.com/amercier/reversi-vue"
        rel="nofollow noopener noreferrer"
        target="_blank"
      >
        Github
      </a>
      .
    
    </p>
     
    <h2
      class="about-subtitle"
    >
      Credits
    </h2>
     
    <p
      class="about-paragraph"
    >
      
      Seamless wood textures by
      
      <a
        class="about-link"
        href="https://www.deviantart.com/siberiancrab"
        rel="nofollow noopener noreferrer"
        target="_blank"
      >
        SiberiaCrab
      </a>
      
      and
      
      <a
        class="about-link"
        href="https://fr.pngtree.com/"
        rel="nofollow noopener noreferrer"
        target="_blank"
      >
        PNG Tree
      </a>
      .
    
    </p>
     
    <p
      class="about-paragraph"
    >
      <a
        class="about-link"
        href="https://www.npmjs.com/package/reversi"
        rel="nofollow noopener noreferrer"
        target="_blank"
      >
        reversi
      </a>
      
      by
      
      <a
        class="about-link"
        href="http://kjirou.net/"
        rel="nofollow noopener noreferrer"
        target="_blank"
      >
        Koujirou Ishii
      </a>
      .
    
    </p>
     
    <p>
      Thank you so much for your hard work!
    </p>
  </div>
</main>
`;
